tests/src/Functional/CommentsGenerationTest.php

```diff
@@ -57,6 +57,21 @@ public function testThereIsNoNewLineAboveTagsSection(\ReflectionClass $mainClass
     {
         $property = $mainClass->getProperty('classmap');
         $this->assertContains('/**
-     * @var array', $property->getDocComment());
+     * @var array', $this->normalizeNEL($property->getDocComment()));
+    }
+
+    /**
+     * Normalizes the newline.
+     *
+     * Windows uses CR+LF for the newline.
+     * This method converts the newline to UNIX style.
+     *
+     * @param string $string
+     *
+     * @return string
+     */
+    private function normalizeNEL($string)
+    {
+        return str_replace("\r", '', $string);
```

Comment:
I see the reasoning behind this but I am wondering whether this is the right way to solve the issue.

As an alternate solution we could [force Linux style line endings using `.gitattributes`](https://help.github.com/articles/dealing-with-line-endings/#per-repository-settings). This seems like a cleaner solution to me. What do you think?